### PR TITLE
fix: add gambhirsharma

### LINKED ISSUE #1298: Gambhir Sharma
## Profile Screenshot

Screenshot 2022-08-12 at 20-10-20 gambhirsharma - Overview

### PR DESCRIPTION
**Pull request by gambhirsharma**
resolves #1298 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/gambhirsharma.json
Comment (Panquesito7) [inline profiles/gambhirsharma.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by gambhirsharma** [comment]
Can I close this issue?

**Comment by AvidCoder101** [comment]
@gambhirsharma no, don't close the issue as the PR needs to be merged first. After the PR has been merged, then a maintainer will close the issue. Hope that helps.

**Review by AvidCoder101 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**
